app_spec/test/regressions.js

```diff
@@ -1,6 +1,6 @@
 module.exports = scenario => {
 
-scenario('calling get_links before link_entries makes no difference', async (s, t, {alice}) => {
+scenario.only('calling get_links before link_entries makes no difference', async (s, t, {alice}) => {
```

Comment:
Looks like you committed your test of this by accident?

```suggestion
scenario('calling get_links before link_entries makes no difference', async (s, t, {alice}) => {
```

Reply:
Let's see if the registerer catches that in CI

Reply:
Well, unfortunately it didn't...that registerer sure has room for improvement